spec/menu_spec.rb

```diff
@@ -0,0 +1,7 @@
+require "menu"
+
+describe Menu do
+  describe "#displaymenu" do
+    it { should_not be_kind_of(String) }
```

Comment:
This doesn't feel like a useful test. Firstly, it's testing state not behaviour. Secondly, asserting that it's not a string kind of means it could be anything else, which doesn't feel like what we want to test. 

Same goes for other tests below.